lib/BloqadeCUDA/src/patch.jl

```diff
@@ -10,3 +11,17 @@ for SparseMatrixType in [:CuSparseMatrixCSC, :CuSparseMatrixCSR]
         end
     end
 end
+
+end
+
+function CuYao.cu(reg::SubspaceArrayReg{D}) where {D}
```

Comment:
I think `SubspaceArrayReg` already overloaded `adapt` thus `adapt(CuArray, reg)` should just work?

Reply:
I might have missed that